### PR TITLE
Always run OOP process on .net core (part 2)

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Followup to https://github.com/dotnet/roslyn/pull/72772.  Removes non-core descriptors as well.  Not sure if this will succeed though given that some of our tests may run on netfx.

### CODE REVIEW

File: src/EditorFeatures/Core/ExternalAccess/UnitTesting/Api/UnitTestingGlobalOptions.cs [hunk 1]
```diff
@@ -5,18 +5,15 @@
...
-using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Remote;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api;
 
 [Export(typeof(UnitTestingGlobalOptions)), Shared]
 [method: ImportingConstructor]
 [method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-internal sealed class UnitTestingGlobalOptions(IGlobalOptionService globalOptions)
+internal sealed class UnitTestingGlobalOptions()
```
Comment (genlu) [inline src/EditorFeatures/Core/ExternalAccess/UnitTesting/Api/UnitTestingGlobalOptions.cs:14]: FYI @shyamnamboodiripad you can remove the check from test window side now

**Review by genlu — APPROVED** [review]
:shipit:

**Comment by davidwengier** [comment]
Does this mean there is no need for non-core service.json files? I'm just working on updating Razor to auto-generate them, but perhaps we need a lot less now?

**Comment by CyrusNajmabadi** [comment]
@jasonmalinowski 	For review when you get back.

**Pull request merged**